From 2ac6b6b084a877adde64db7faff2ed22eb3ea97a Mon Sep 17 00:00:00 2001
From: Daniel Stone <daniels@collabora.com>
Date: Tue, 8 Feb 2022 22:39:42 +0000
Subject: [PATCH] tests: Add dependency on screenshooter client protocol

Given that the test-helper code relies on the screenshooter protocol,
make sure it's available for us to build, and the dependency ensures we
build in order.

Fixes: #588

Signed-off-by: Daniel Stone <daniels@collabora.com>

[Retrieved from:
https://gitlab.freedesktop.org/wayland/weston/-/commit/2ac6b6b084a877adde64db7faff2ed22eb3ea97a]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 tests/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index 2d464ddcc..222091cd1 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -29,8 +29,9 @@ lib_test_client = static_library(
 		'weston-test-client-helper.c',
 		'weston-test-fixture-compositor.c',
 		weston_test_client_protocol_h,
-		weston_screenshooter_protocol_c,
 		weston_test_protocol_c,
+		weston_screenshooter_client_protocol_h,
+		weston_screenshooter_protocol_c,
 		viewporter_client_protocol_h,
 		viewporter_protocol_c,
 		'color_util.h',
-- 
GitLab

